exercises/list-ops/list_ops_test.rb

```diff
@@ -0,0 +1,107 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'list_ops'
+
+class ListOpsTest < Minitest::Test
+  def test_count_empty
+    assert_equal 0, ListOps.arrays([])
+  end
+
+  def test_count_normal
+    skip
+    assert_equal 5, ListOps.arrays(Array.new(5))
+  end
+
+  def test_count_gigantic
+    skip
+    assert_equal 1_000_000, ListOps.arrays(Array.new(1_000_000))
+  end
+
+  def test_reverse_empty
+    skip
+    assert_equal [], ListOps.reverser([])
+  end
+
+  def test_reverse_normal
```

Comment:
I think we have the same situation here regarding the word `normal`, what is it in this context? There are a few tests like this, maybe we could change them so the title becomes more expressive?